images/builders/mlflow/README.md

```diff
@@ -0,0 +1,146 @@
+    type: string
+steps:
+  - name: builder
+    image: ghcr.io/fuseml/mlflow-builder:latest
+    inputs:
+      - name: mlflow-codeset
+        codeset:
+          name: '{{ inputs.mlflow-codeset }}'
+          path: /project
+    outputs:
+      - name: mlflow-runtime-image
+```
+
+Aside from the mandatory codeset input, the MLflow builder workflow step also accepts the following optional input parameters that can be used to customize how the ML python environment container image is being built and published:
+
+- `registry` - the container registry hostname that is used by the builder step when pushing the built MLflow environment image to the container registry. The default value is `registry.fuseml-registry`, which points to the FuseML builtin container registry. 
+- `pull_registry` - the container registry hostname used by consumers that need to pull the built MLflow environment image (e.g. the container runtime instances running on the Kubernetes cluster nodes where the FuseML workflow is executed). In most cases, this is the same as the `registry` parameter. However, sometimes, this value needs to be different than `registry`, for example when the container registry is accessed through different endpoints by the builder step and by the kubernetes cluster itself. In particular, when using the default internal FuseML registry, we need to reference the repository using the node's localhost address (see https://github.com/fuseml/fuseml/issues/65). The default value is `127.0.0.1:30500`, which is how the FuseML builtin container registry is exposed internally to the Kubernetes cluster. If not set, or set to an empty value, the `registry` parameter value will be used instead.
+- `repository` - the container registry repository name that is used by the builder step when pushing the built MLflow environment image to the container registry. The default value is `mlflow/trainer`.
+- `miniconda_version` - the version (tag) of the `continuumio/miniconda3` base container image to use when building MLflow environment container images based on conda (i.e. when a `conda.yaml` file is included in the codeset). If not specified, the builder will default to `4.10.3`.
+- `base_image` - the base image to use when building MLflow environment container images based on pip (i.e. when a pip `requirements.txt` file is included in the codeset). If not specified, the builder will default to `python:3.6.13`.
+- `verbose` - set to `true` to enable verbose logging in the builder workflow step (default is `false`).
+- `compressed_caching` - use this parameter to control the compressed caching feature used by [Kaniko](https://github.com/GoogleContainerTools/kaniko) when building container images. Compressed caching is set to false by default to reduce memory usage with larger images and avoid OOM problems. Set this flag to true to enable compressed caching and reduce the image build time.
+
+
+## Using the MLflow Runtime Environment Step
```

Comment:
While "MLflow Runtime Environment Step" is a great title, it's only at the very end of this section that you mention it as a "trainer" step. I think the actual work that this step is doing (and it _is_ calling `mlflow train` after all) should be mentioned right at the top.

Reply:
I kept the current title but I added a note mentioning the that most common use-case for the environment step is to train ML models and that's why it's also being referred to as a "trainer" step